app/common/constants/ledgerMediaProviders.js

```diff
@@ -0,0 +1,12 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const mapValuesByKeys = require('../../../js/lib/functional').mapValuesByKeys
```

Comment:
I believe there is a performance impact by using this... Going forward, I think it might be better to just assign a value directly.  For example:
```js
const providers = {
  YOUTUBE: 'youtube.com',
  TWITCH: 'twitch.tv'
  //... etc ...
}
```

Reply:
cc: @bbondy

Reply:
I used what we have now for other constants, but I can change it

Reply:
replaced with string